CHANGELOG.asciidoc

```diff
@@ -20,7 +20,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Use `initial_scan` action for new paths. {pull}7954[7954]
 
 *Filebeat*
-
```

Comment:
Nit: You've removed the empty line between the header (Filebeat) and the list of changes

Reply:
Will fix this in a follow up PR